tools/clang/test/CodeGenSPIRV_Lit/vk.layout.shader-record-ext.std430.hlsl

```diff
@@ -44,7 +44,7 @@ struct S {
 [[vk::shader_record_ext]]
 ConstantBuffer<S> cbuf;
```

Comment:
I'm a bit confused, this test is supposed to only use the KHR variant of the ShaderRecordBuffer extensions?
Building this test with the DXC at this PR ends up generating this kind of lines:
```
 %_ptr_ShaderRecordBufferNV_type_ShaderRecordBufferKHR_block = OpTypePointer ShaderRecordBufferNV %type_ShaderRecordBufferKHR_block

...

%block = OpVariable %_ptr_ShaderRecordBufferNV_type_ShaderRecordBufferKHR_block ShaderRecordBufferNV
```

Aren't the NV version not supposed to be used?

Reply:
The NV and KHR constants have the same value; the SPIRV-Tools disassembler is defaulting to using the NV string for that constant.